### ISSUE TITLE
[Bug?] Hostile ships using lightcruise & disappearing [20w17e]

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Starship-EVO#1864]
### Short Description of the issue:
Not entirely sure this is a bug, because running away when it's losing would actually be pretty smart of the AI, but just in case:

The NPC ships are apparently using lightcruise to jump around during combat. They will leave sensor range and re-enter it. The last time this happened, the ship in question left and never came back.

They will also sometimes make microjumps, where they jump only a couple kilometers, basically changing their position instantly while not leaving combat range. This is very disorienting for a human opponent.

### Steps to reproduce the issue:
Spawn a hostile ship and fight it for a while. Not sure precisely what causes this to happen, but it keeps happening for me sooner or later.

### Environment
- Skywanderers version: 20w17e
- Graphics Card: NVIDIA GeForce GTX 1060 6GB

### Additional information:
You can see the hostile ship here on the left side of the screen. It's far enough away that the white dot thing in the middle of the target box completely covers it.
![20200426014318_1](https://user-images.githubusercontent.com/33041786/80300130-95050b00-875f-11ea-8c2d-9eca5e6e0986.jpg)

This second screenshot was taken about a second or two later; you can see how fast it's moving by how far it's gone since the first shot.
![20200426014321_2](https://user-images.githubusercontent.com/33041786/80300128-91718400-875f-11ea-9553-ac9cb6c9f58f.jpg)

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4534755/output_log.txt)

**Comment by tsunamayo** [comment]
Yes they try to fly, and they will use lightcruise to get to you. They should not use lightcruise (yet?) to flee though, so I need to investigate. 
Also yes they flee too much...
I keep you posted.
